python/ml4ir/base/data/relevance_dataset.py

```diff
@@ -165,6 +165,12 @@ def create_dataset(self, parse_tfrecord=True):
             as it improved train/test/validation throughput
             by 30% in some real model training.
             """
+            # Setting the one-hot mask of the fr feature to be off for training
```

Comment:
what does `fr` stand for?

Reply:
final rank. It's one of the logged features indicating at which rank this document was shown